crates/next-core/src/next_client/context.rs

```diff
@@ -98,6 +98,7 @@ pub async fn get_client_module_options_context(
         // we try resolve it once at the root and pass down a context to all
         // the modules.
         enable_emotion: true,
+        enable_next_font: true,
```

Comment:
This only enables next font on client side, but it should probably also work on server side for RSC or SSR